### ISSUE TITLE
Issues in KeyError: 'novel'

### ISSUE THREAD

**Issue by kerrwelton** [ablab/IsoQuant#245]
Hi, thank you for developing this excellent software. I encountered an issue while running isoquant.py to analyze Oxford Nanopore  data. Could you please let me know how to resolve it? I would greatly appreciate it.

Here are the parameters I used when running isoquant.py.
isoquant.py --reference GRCh38.p14.genome.fa \
                     --genedb gencode.v46.annotation.gtf --complete_genedb \
                     --bam memcd4.bam ncd4.bam ncd8.bam --labels memcd4 ncd4 ncd8 \
                     --data_type nanopore \
                     -o isoquant_result \
                     --threads 80 \
                     --check_canonical

The error message when running isoquant.py was saved in nohup.out.log, isoquant.log is the automatically generated by isoquant.py.
[isoquant.log](https://github.com/user-attachments/files/17128199/isoquant.log)
[nohup.out.log](https://github.com/user-attachments/files/17128216/nohup.out.log)

**Comment by andrewprzh** [comment]
Hi @kerrwelton 

Thanks for the feedback! Which version are you using? It seems I've seen issue in one of the previous IsoQuant versions and had been fixed since then.

Best
Andrey

**Comment by kerrwelton** [comment]
I am using isoquant v2.2.0. Do I need to upgrade to a higher version?

**Comment by andrewprzh** [comment]
Yes, 2.2 is more than 2 years old, current version is 3.6, I highly recommend updating.

**Comment by kerrwelton** [comment]
I updated isoquant to the latest version and rerun isoquant.py. Now it runs successfully with no error message. Thanks for you rapid and helpful response. :) :)
